pkg/session/serve.go

```diff
@@ -103,39 +104,33 @@ func (s *Session) serve() {
 			for _, componentName := range payload.Components {
```

Comment:
we should refactor the for loop:
1. wrap it as a function
2. get component by name, and check if the component implements SetHealthy method, if so, call it, otherwise skip it.

Reply:
This requires `unwrap` method since `getComponent` itself returns interface... and `unwrap` is being removed as we remove watchable component interface (since it's too complicated). Do we want to add back?

Reply:
I don't think we need the `Unwrap` method even `getComponent` returns interface since type assert checks if the underlay type supports target interface.

Reply:
Yeah understood your comment now, fixed.